cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -490,15 +487,17 @@ define([
         });
     });
 
-    describe('GroupConfigurationsList', function() {
-        var emptyMessage = 'You haven\'t created any group configurations yet.';
+    describe('GroupConfigurationsListView', function() {
```

Comment:
As mentioned above, make it clear which tests are for experiment group configurations.